src/searchbar.vue

```diff
@@ -0,0 +1,52 @@
+<script>
+    import Typeahead from './Typeahead.vue'
+    export default {
+      extends: Typeahead,
+      computed: {
+        primitiveData() {
+          if (this.data) {
+            var matches = []
+            var words
+            this.data.map(value => {
+              var query = this.value.toLowerCase()
```

Comment:
Instead of having to convert to lower case `toLowerCase()`, maybe we can use `search()` instead of `index()`?

So instead of: `heading.indexOf(query)`, we use `heading.search(new RegExp(query, "i")))`. The "i" will make the search case-insensitive.

Hopefully this will make the code clearer.

Reply:
@nusjzx Is it necessary to modify `primitiveData`?